code/modules/ruins/lavalandruin_code/dead_ratvar.dm

```diff
@@ -258,3 +258,8 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	w_class = WEIGHT_CLASS_BULKY
 	var/bonus_burn = 5
+
+/obj/item/clockwork/weapon/ratvarian_spear/attack(mob/living/M, mob/living/user, def_zone)
+	. = ..()
+	if(isliving(M))
```

Comment:
Might want to check the return result here so that stuff like pacifism works properly

Reply:
right, I hadn't considered pacifism, will look into this.